### PR TITLE
add Built with Spike resource

### PR DESCRIPTION
**Pull request by tkraak**

### CODE REVIEW

**Comment by jescalan** [comment]
Drop a screenshot in here!

**Comment by tkraak** [comment]
built-with-spike

**Comment by tkraak** [comment]
Typo ... hold on.

File: views/index.sgr [hunk 1]
```diff
@@ -29,7 +29,11 @@ extends(src='layout.sgr')
           .text
             h4 Community
             p Come on over to our gitter chatroom, where a community of Spike user can help out. Or just chat about how great life is.
-
+        li: a(href='https://spike.readme.io/docs/built-with-spike')
+          .icon.built-with-spike
+          .text
+            h4 Build with Spike
```
Comment (jescalan) [inline views/index.sgr]: buil**t** with spike?

**Comment by jescalan** [comment]
Beautiful ✨ -- thanks!

**Pull request merged**